src/Shared/StellarChat.Shared.Infrastructure/Identity/Extensions.cs

```diff
@@ -0,0 +1,60 @@
+﻿using AspNetCore.Identity.MongoDbCore.Extensions;
+using AspNetCore.Identity.MongoDbCore.Infrastructure;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+using StellarChat.Shared.Infrastructure.DAL.Mongo;
+using StellarChat.Shared.Infrastructure.Identity.Seeders;
+
+namespace StellarChat.Shared.Infrastructure.Identity;
+
+public static class Extensions
+{  
+    public static IServiceCollection AddMongoIdentity(this IServiceCollection services, IConfiguration configuration)
+    {
+        var serviceProvider = services.BuildServiceProvider();
+        var options = serviceProvider.GetRequiredService<IOptions<MongoOptions>>().Value;
+
+        if (options == null) 
+            return services; 
+
+        var mongoDbIdentityConfig = new MongoDbIdentityConfiguration
+        {
+            MongoDbSettings = new MongoDbSettings
+            { 
+                ConnectionString = options.CONNECTION_STRING,
+                DatabaseName = "StellarChat"
```

Comment:
Avoid hardcoding values directly in the code. Instead, consider using constant variables or values from appsettings.json. For example, the database name can be retrieved from the **mongo:database** setting in [appsettings.json](https://github.com/ktutak1337/Stellar-Chat/blob/main/src/Server/StellarChat.Server.Api/appsettings.json). This approach will make the code more maintainable and configurable.